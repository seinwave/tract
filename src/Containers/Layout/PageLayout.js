import React, { useState } from "react";
import SideBar from "../SideBar/SideBar";
import TopBar from "../TopBar/TopBar";
import FinancialPage from "../../Components/Pages/FinancialPage/FinancialPage";
import TeamPage from "../../Components/Pages/TeamPage/TeamPage";
import TicketPage from "../../Components/Pages/TicketPage/TicketPage";
import ProjectPage from "../../Components/Pages/ProjectPage/ProjectPage";
import Dashboard from "../../Components/Pages/Dashboard/Dashboard";
import { Layout, Menu, Breadcrumb } from "antd";
import {
  BrowserRouter as Router,
  Route,
  NavLink,
  Switch,
} from "react-router-dom";
import {
  PieChartOutlined,
  SettingOutlined,
  DollarOutlined,
  InboxOutlined,
  TeamOutlined,
  BookOutlined,
  UserOutlined,
} from "@ant-design/icons";

import "./PageLayout.scss";
import "antd/dist/antd.css";

const { Header, Footer, Sider, Content } = Layout;
const { SubMenu } = Menu;

function PageLayout() {
  const [collapsed, setCollapsed] = useState(false);

  const onCollapse = () => {
    setCollapsed(!collapsed);
  };

  return (
    <>
      <Router>
        <Layout theme="light" hasSider={true}>
          <Sider
            theme="light"
            collapsible
            collapsed={collapsed}
            onCollapse={onCollapse}
            style={{
              overflow: "auto",
              height: "100vh",
              position: "fixed",
              left: 0,
            }}
          >
            <div className="logoContainer">
              <div className="logo">
                <img
                  src="src/images/logo192.png"
                  className="mr-2"
                  alt="Clean UI"
                />
                <div className="name">Tract</div>
              </div>
            </div>
            <Menu defaultSelectedKeys={["1"]} mode="inline">
              <Menu.Item key="1" icon={<PieChartOutlined />}>
                <NavLink className="nav-item" to="/home">
                  Home
                </NavLink>
              </Menu.Item>
              <Menu.Item key="2" icon={<TeamOutlined />}>
                <NavLink className="nav-item" to="/team">
                  Team
                </NavLink>
              </Menu.Item>
              <Menu.Item key="3" icon={<BookOutlined />}>
                <NavLink className="nav-item" to="/projects">
                  Projects
                </NavLink>
              </Menu.Item>
              <Menu.Item key="4" icon={<InboxOutlined />}>
                <NavLink className="nav-item" to="/tickets">
                  Tickets
                </NavLink>
              </Menu.Item>
              <Menu.Item key="5" icon={<DollarOutlined />}>
                <NavLink className="nav-item" to="/financials">
                  Financials
                </NavLink>
              </Menu.Item>

              <div class="h-divider"></div>
              <Menu.Item key="6" icon={<UserOutlined />}>
                <NavLink className="nav-item" to="/profile">
                  Profile
                </NavLink>
              </Menu.Item>
              <Menu.Item key="7" icon={<SettingOutlined />}>
                <NavLink className="nav-item" to="/exit">
                  Sign Out
                </NavLink>
              </Menu.Item>
            </Menu>
          </Sider>
          <Layout theme="light">
            <Header id="real-header" theme="light">
              <TopBar />
            </Header>
            <Content>
              <Switch>
                <Route path="/financials">
                  <FinancialPage />
                </Route>
                <Route path="/home">
                  <Dashboard />
                </Route>
                <Route path="/projects">
                  <ProjectPage />
                </Route>
                <Route path="/team">
                  <TeamPage />
                </Route>
                <Route path="/tickets">
                  <TicketPage />
                </Route>
              </Switch>
            </Content>
          </Layout>
        </Layout>
      </Router>
    </>
  );
}

export default PageLayout;
