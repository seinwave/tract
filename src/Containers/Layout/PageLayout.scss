#real-header {
    background: #fff;
    padding: 0;
    height: 87px; 
}

.logoContainer {
    height: 84px;
}

.logo {
    padding: 15px;
}
.corner-logo {
    height: 50px;
}

.name {
    font-weight: bold;
    padding-left: 15px;
    font-size: 20px;
    position: absolute;
    top: 25px;
}

// #page-content-wrapper {
//     padding: 1rem;
//     margin: 1rem;
// }

// .fa {
//     text-align: right;
//     padding-left: 2rem;
// }

// .menu-icon{
//     text-align: center;
//     position: absolute;
//     right: 14px;
//     width: 20px;
//     top: 4px;
//     margin: 0 !important;
//     line-height: 1 !important;
//     font-size: 1.13rem;
//     transition: color .3s cubic-bezier(.645,.045,.355,1);
// }

// .nav-row {
//     display: flex;
//     flex-direction: row;
//     justify-content: space-between;
//     align-items: stretch;
// }

// .topbar-wrapper {
//     background-color: white;
// }

.h-divider {
    margin-top: 5px;
    margin-bottom: 5px;
    margin-left: 15%;
    height: 1px;
    width: 70%;
    border-top: 1px solid gray;
}

