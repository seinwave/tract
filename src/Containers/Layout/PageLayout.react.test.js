import React from 'react';
import PageLayout from './PageLayout'
import renderer from 'react-test-renderer'

it('renders as expected', () => {
    expect(2).toEqual(2);

    // const tree = renderer
    // .create(<PageLayout/>)
    // .toJSON();
    // expect(tree).toMatchSnapshot(); 
})