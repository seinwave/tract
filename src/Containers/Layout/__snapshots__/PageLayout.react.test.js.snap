// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<section
  className="ant-layout ant-layout-has-sider"
  theme="light"
>
  <aside
    className="ant-layout-sider ant-layout-sider-light ant-layout-sider-has-trigger"
    style={
      Object {
        "flex": "0 0 200px",
        "height": "100vh",
        "left": 0,
        "maxWidth": "200px",
        "minWidth": "200px",
        "overflow": "auto",
        "position": "fixed",
        "width": "200px",
      }
    }
  >
    <div
      className="ant-layout-sider-children"
    >
      <div
        className="logoContainer"
      >
        <div
          className="logo"
        >
          <img
            alt="logo"
            className="corner-logo"
            src="https://upload.wikimedia.org/wikipedia/en/thumb/3/3b/Chipotle_Mexican_Grill_logo.svg/1200px-Chipotle_Mexican_Grill_logo.svg.png"
          />
          <span
            className="name"
          >
            Tract
          </span>
        </div>
      </div>
      <ul
        className="ant-menu ant-menu-light ant-menu-root ant-menu-inline"
        onMouseEnter={[Function]}
        onTransitionEnd={[Function]}
        role="menu"
        style={Object {}}
      >
        <li
          className="ant-menu-item ant-menu-item-selected"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="menuitem"
          style={
            Object {
              "paddingLeft": 24,
            }
          }
        >
          <span
            aria-label="pie-chart"
            className="anticon anticon-pie-chart"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="pie-chart"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M864 518H506V160c0-4.4-3.6-8-8-8h-26a398.46 398.46 0 00-282.8 117.1 398.19 398.19 0 00-85.7 127.1A397.61 397.61 0 0072 552a398.46 398.46 0 00117.1 282.8c36.7 36.7 79.5 65.6 127.1 85.7A397.61 397.61 0 00472 952a398.46 398.46 0 00282.8-117.1c36.7-36.7 65.6-79.5 85.7-127.1A397.61 397.61 0 00872 552v-26c0-4.4-3.6-8-8-8zM705.7 787.8A331.59 331.59 0 01470.4 884c-88.1-.4-170.9-34.9-233.2-97.2C174.5 724.1 140 640.7 140 552c0-88.7 34.5-172.1 97.2-234.8 54.6-54.6 124.9-87.9 200.8-95.5V586h364.3c-7.7 76.3-41.3 147-96.6 201.8zM952 462.4l-2.6-28.2c-8.5-92.1-49.4-179-115.2-244.6A399.4 399.4 0 00589 74.6L560.7 72c-4.7-.4-8.7 3.2-8.7 7.9V464c0 4.4 3.6 8 8 8l384-1c4.7 0 8.4-4 8-8.6zm-332.2-58.2V147.6a332.24 332.24 0 01166.4 89.8c45.7 45.6 77 103.6 90 166.1l-256.4.7z"
              />
            </svg>
          </span>
          <span>
            <a
              aria-current={null}
              className="nav-item"
              href="/home"
              onClick={[Function]}
            >
              Home
            </a>
          </span>
        </li>
        <li
          className="ant-menu-item"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="menuitem"
          style={
            Object {
              "paddingLeft": 24,
            }
          }
        >
          <span
            aria-label="team"
            className="anticon anticon-team"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="team"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M824.2 699.9a301.55 301.55 0 00-86.4-60.4C783.1 602.8 812 546.8 812 484c0-110.8-92.4-201.7-203.2-200-109.1 1.7-197 90.6-197 200 0 62.8 29 118.8 74.2 155.5a300.95 300.95 0 00-86.4 60.4C345 754.6 314 826.8 312 903.8a8 8 0 008 8.2h56c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5A226.62 226.62 0 01612 684c60.9 0 118.2 23.7 161.3 66.8C814.5 792 838 846.3 840 904.3c.1 4.3 3.7 7.7 8 7.7h56a8 8 0 008-8.2c-2-77-33-149.2-87.8-203.9zM612 612c-34.2 0-66.4-13.3-90.5-37.5a126.86 126.86 0 01-37.5-91.8c.3-32.8 13.4-64.5 36.3-88 24-24.6 56.1-38.3 90.4-38.7 33.9-.3 66.8 12.9 91 36.6 24.8 24.3 38.4 56.8 38.4 91.4 0 34.2-13.3 66.3-37.5 90.5A127.3 127.3 0 01612 612zM361.5 510.4c-.9-8.7-1.4-17.5-1.4-26.4 0-15.9 1.5-31.4 4.3-46.5.7-3.6-1.2-7.3-4.5-8.8-13.6-6.1-26.1-14.5-36.9-25.1a127.54 127.54 0 01-38.7-95.4c.9-32.1 13.8-62.6 36.3-85.6 24.7-25.3 57.9-39.1 93.2-38.7 31.9.3 62.7 12.6 86 34.4 7.9 7.4 14.7 15.6 20.4 24.4 2 3.1 5.9 4.4 9.3 3.2 17.6-6.1 36.2-10.4 55.3-12.4 5.6-.6 8.8-6.6 6.3-11.6-32.5-64.3-98.9-108.7-175.7-109.9-110.9-1.7-203.3 89.2-203.3 199.9 0 62.8 28.9 118.8 74.2 155.5-31.8 14.7-61.1 35-86.5 60.4-54.8 54.7-85.8 126.9-87.8 204a8 8 0 008 8.2h56.1c4.3 0 7.9-3.4 8-7.7 1.9-58 25.4-112.3 66.7-153.5 29.4-29.4 65.4-49.8 104.7-59.7 3.9-1 6.5-4.7 6-8.7z"
              />
            </svg>
          </span>
          <span>
            <a
              aria-current={null}
              className="nav-item"
              href="/team"
              onClick={[Function]}
            >
              Team
            </a>
          </span>
        </li>
        <li
          className="ant-menu-item"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="menuitem"
          style={
            Object {
              "paddingLeft": 24,
            }
          }
        >
          <span
            aria-label="book"
            className="anticon anticon-book"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="book"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M832 64H192c-17.7 0-32 14.3-32 32v832c0 17.7 14.3 32 32 32h640c17.7 0 32-14.3 32-32V96c0-17.7-14.3-32-32-32zm-260 72h96v209.9L621.5 312 572 347.4V136zm220 752H232V136h280v296.9c0 3.3 1 6.6 3 9.3a15.9 15.9 0 0022.3 3.7l83.8-59.9 81.4 59.4c2.7 2 6 3.1 9.4 3.1 8.8 0 16-7.2 16-16V136h64v752z"
              />
            </svg>
          </span>
          <span>
            <a
              aria-current={null}
              className="nav-item"
              href="/projects"
              onClick={[Function]}
            >
              Projects
            </a>
          </span>
        </li>
        <li
          className="ant-menu-item"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="menuitem"
          style={
            Object {
              "paddingLeft": 24,
            }
          }
        >
          <span
            aria-label="inbox"
            className="anticon anticon-inbox"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="inbox"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="0 0 1024 1024"
              width="1em"
            >
              <path
                d="M885.2 446.3l-.2-.8-112.2-285.1c-5-16.1-19.9-27.2-36.8-27.2H281.2c-17 0-32.1 11.3-36.9 27.6L139.4 443l-.3.7-.2.8c-1.3 4.9-1.7 9.9-1 14.8-.1 1.6-.2 3.2-.2 4.8V830a60.9 60.9 0 0060.8 60.8h627.2c33.5 0 60.8-27.3 60.9-60.8V464.1c0-1.3 0-2.6-.1-3.7.4-4.9 0-9.6-1.3-14.1zm-295.8-43l-.3 15.7c-.8 44.9-31.8 75.1-77.1 75.1-22.1 0-41.1-7.1-54.8-20.6S436 441.2 435.6 419l-.3-15.7H229.5L309 210h399.2l81.7 193.3H589.4zm-375 76.8h157.3c24.3 57.1 76 90.8 140.4 90.8 33.7 0 65-9.4 90.3-27.2 22.2-15.6 39.5-37.4 50.7-63.6h156.5V814H214.4V480.1z"
              />
            </svg>
          </span>
          <span>
            <a
              aria-current={null}
              className="nav-item"
              href="/tickets"
              onClick={[Function]}
            >
              Tickets
            </a>
          </span>
        </li>
        <li
          className="ant-menu-item"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="menuitem"
          style={
            Object {
              "paddingLeft": 24,
            }
          }
        >
          <span
            aria-label="dollar"
            className="anticon anticon-dollar"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="dollar"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372zm47.7-395.2l-25.4-5.9V348.6c38 5.2 61.5 29 65.5 58.2.5 4 3.9 6.9 7.9 6.9h44.9c4.7 0 8.4-4.1 8-8.8-6.1-62.3-57.4-102.3-125.9-109.2V263c0-4.4-3.6-8-8-8h-28.1c-4.4 0-8 3.6-8 8v33c-70.8 6.9-126.2 46-126.2 119 0 67.6 49.8 100.2 102.1 112.7l24.7 6.3v142.7c-44.2-5.9-69-29.5-74.1-61.3-.6-3.8-4-6.6-7.9-6.6H363c-4.7 0-8.4 4-8 8.7 4.5 55 46.2 105.6 135.2 112.1V761c0 4.4 3.6 8 8 8h28.4c4.4 0 8-3.6 8-8.1l-.2-31.7c78.3-6.9 134.3-48.8 134.3-124-.1-69.4-44.2-100.4-109-116.4zm-68.6-16.2c-5.6-1.6-10.3-3.1-15-5-33.8-12.2-49.5-31.9-49.5-57.3 0-36.3 27.5-57 64.5-61.7v124zM534.3 677V543.3c3.1.9 5.9 1.6 8.8 2.2 47.3 14.4 63.2 34.4 63.2 65.1 0 39.1-29.4 62.6-72 66.4z"
              />
            </svg>
          </span>
          <span>
            <a
              aria-current={null}
              className="nav-item"
              href="/financials"
              onClick={[Function]}
            >
              Expenses
            </a>
          </span>
        </li>
        <div
          class="h-divider"
        />
        <li
          className="ant-menu-item"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="menuitem"
          style={
            Object {
              "paddingLeft": 24,
            }
          }
        >
          <span
            aria-label="user"
            className="anticon anticon-user"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="user"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M858.5 763.6a374 374 0 00-80.6-119.5 375.63 375.63 0 00-119.5-80.6c-.4-.2-.8-.3-1.2-.5C719.5 518 760 444.7 760 362c0-137-111-248-248-248S264 225 264 362c0 82.7 40.5 156 102.8 201.1-.4.2-.8.3-1.2.5-44.8 18.9-85 46-119.5 80.6a375.63 375.63 0 00-80.6 119.5A371.7 371.7 0 00136 901.8a8 8 0 008 8.2h60c4.4 0 7.9-3.5 8-7.8 2-77.2 33-149.5 87.8-204.3 56.7-56.7 132-87.9 212.2-87.9s155.5 31.2 212.2 87.9C779 752.7 810 825 812 902.2c.1 4.4 3.6 7.8 8 7.8h60a8 8 0 008-8.2c-1-47.8-10.9-94.3-29.5-138.2zM512 534c-45.9 0-89.1-17.9-121.6-50.4S340 407.9 340 362c0-45.9 17.9-89.1 50.4-121.6S466.1 190 512 190s89.1 17.9 121.6 50.4S684 316.1 684 362c0 45.9-17.9 89.1-50.4 121.6S557.9 534 512 534z"
              />
            </svg>
          </span>
          <span>
            <a
              aria-current={null}
              className="nav-item"
              href="/profile"
              onClick={[Function]}
            >
              Profile
            </a>
          </span>
        </li>
        <li
          className="ant-menu-item"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          role="menuitem"
          style={
            Object {
              "paddingLeft": 24,
            }
          }
        >
          <span
            aria-label="setting"
            className="anticon anticon-setting"
            role="img"
          >
            <svg
              aria-hidden="true"
              data-icon="setting"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M924.8 625.7l-65.5-56c3.1-19 4.7-38.4 4.7-57.8s-1.6-38.8-4.7-57.8l65.5-56a32.03 32.03 0 009.3-35.2l-.9-2.6a443.74 443.74 0 00-79.7-137.9l-1.8-2.1a32.12 32.12 0 00-35.1-9.5l-81.3 28.9c-30-24.6-63.5-44-99.7-57.6l-15.7-85a32.05 32.05 0 00-25.8-25.7l-2.7-.5c-52.1-9.4-106.9-9.4-159 0l-2.7.5a32.05 32.05 0 00-25.8 25.7l-15.8 85.4a351.86 351.86 0 00-99 57.4l-81.9-29.1a32 32 0 00-35.1 9.5l-1.8 2.1a446.02 446.02 0 00-79.7 137.9l-.9 2.6c-4.5 12.5-.8 26.5 9.3 35.2l66.3 56.6c-3.1 18.8-4.6 38-4.6 57.1 0 19.2 1.5 38.4 4.6 57.1L99 625.5a32.03 32.03 0 00-9.3 35.2l.9 2.6c18.1 50.4 44.9 96.9 79.7 137.9l1.8 2.1a32.12 32.12 0 0035.1 9.5l81.9-29.1c29.8 24.5 63.1 43.9 99 57.4l15.8 85.4a32.05 32.05 0 0025.8 25.7l2.7.5a449.4 449.4 0 00159 0l2.7-.5a32.05 32.05 0 0025.8-25.7l15.7-85a350 350 0 0099.7-57.6l81.3 28.9a32 32 0 0035.1-9.5l1.8-2.1c34.8-41.1 61.6-87.5 79.7-137.9l.9-2.6c4.5-12.3.8-26.3-9.3-35zM788.3 465.9c2.5 15.1 3.8 30.6 3.8 46.1s-1.3 31-3.8 46.1l-6.6 40.1 74.7 63.9a370.03 370.03 0 01-42.6 73.6L721 702.8l-31.4 25.8c-23.9 19.6-50.5 35-79.3 45.8l-38.1 14.3-17.9 97a377.5 377.5 0 01-85 0l-17.9-97.2-37.8-14.5c-28.5-10.8-55-26.2-78.7-45.7l-31.4-25.9-93.4 33.2c-17-22.9-31.2-47.6-42.6-73.6l75.5-64.5-6.5-40c-2.4-14.9-3.7-30.3-3.7-45.5 0-15.3 1.2-30.6 3.7-45.5l6.5-40-75.5-64.5c11.3-26.1 25.6-50.7 42.6-73.6l93.4 33.2 31.4-25.9c23.7-19.5 50.2-34.9 78.7-45.7l37.9-14.3 17.9-97.2c28.1-3.2 56.8-3.2 85 0l17.9 97 38.1 14.3c28.7 10.8 55.4 26.2 79.3 45.8l31.4 25.8 92.8-32.9c17 22.9 31.2 47.6 42.6 73.6L781.8 426l6.5 39.9zM512 326c-97.2 0-176 78.8-176 176s78.8 176 176 176 176-78.8 176-176-78.8-176-176-176zm79.2 255.2A111.6 111.6 0 01512 614c-29.9 0-58-11.7-79.2-32.8A111.6 111.6 0 01400 502c0-29.9 11.7-58 32.8-79.2C454 401.6 482.1 390 512 390c29.9 0 58 11.6 79.2 32.8A111.6 111.6 0 01624 502c0 29.9-11.7 58-32.8 79.2z"
              />
            </svg>
          </span>
          <span>
            <a
              aria-current={null}
              className="nav-item"
              href="/exit"
              onClick={[Function]}
            >
              Sign Out
            </a>
          </span>
        </li>
      </ul>
    </div>
    <div
      className="ant-layout-sider-trigger"
      onClick={[Function]}
      style={
        Object {
          "width": "200px",
        }
      }
    >
      <span
        aria-label="left"
        className="anticon anticon-left"
        role="img"
      >
        <svg
          aria-hidden="true"
          data-icon="left"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M724 218.3V141c0-6.7-7.7-10.4-12.9-6.3L260.3 486.8a31.86 31.86 0 000 50.3l450.8 352.1c5.3 4.1 12.9.4 12.9-6.3v-77.3c0-4.9-2.3-9.6-6.1-12.6l-360-281 360-281.1c3.8-3 6.1-7.7 6.1-12.6z"
          />
        </svg>
      </span>
    </div>
  </aside>
  <section
    className="ant-layout"
    theme="light"
  >
    <header
      className="ant-layout-header"
      id="real-header"
      theme="light"
    >
      <div
        className="topbar"
      >
        <nav
          className="navbar navbar-expand-lg navbar-light"
        >
          <div
            className="collapse navbar-collapse"
            id="navbarSupportedContent"
          />
          <div
            className="avatar-dropdown-button dropdown"
            onKeyDown={[Function]}
          >
            <button
              aria-expanded={false}
              aria-haspopup={true}
              className="dropdown-toggle btn btn-primary"
              disabled={false}
              onClick={[Function]}
              type="button"
            >
              <img
                alt="Avatarimage"
                className="topbar-avatar-image"
                src="https://cdn.iconscout.com/icon/free/png-256/avatar-375-456327.png"
              />
               
              Hi, USER
            </button>
          </div>
        </nav>
      </div>
    </header>
    <main
      className="ant-layout-content"
    />
  </section>
</section>
`;
